src/main/commandServer/__tests__/httpCommandServer.spec.ts

```diff
@@ -47,16 +47,27 @@ describe(HttpCommandServer, () => {
    * or by developers during a typical edit-test-fix cycle, these are edge cases we can ignore for now.
    */
   itNonWindows("should fail to run rdctl shell when server isn't running", async() => {
-    const listSettingsRejects = expect(() => spawnFile(rdctlPath, ['list-settings'], { stdio: 'pipe' })).rejects;
+    const listSettingsRejects = expect(() => spawnFile(rdctlPath,
+      [
+        'list-settings', '--user=user', '--password=puser', '--port=6107'
```

Comment:
We shouldn't be setting the parameters explicitly here; if we, say, end up changing the port here, it will erroneously pass (i.e. fail to connect) even if we end up listening on the new port.

Reply:
This came from the branch with initial commit "Add a test that verifies how `rdctl shell` fails when server [is] not running."

So we want to determine if rancher-desktop is not running (meaning `rdctl list-settings` works), and run the tests only if it isn't, because we're interested in the error message when it isn't (which brings in another can of worms).

The tests only run if rancher-desktop isn't running, so it actually doesn't matter which port we hit. I specify those fields so the test can run on a CI or post-factory-reset where there's no `rd-engine.json` file with the actual connection fields.

I'll make a change to make this more obvious.